TeamCode/src/main/java/org/firstinspires/ftc/teamcode/opmodes/BasicTeleOp.java

```diff
@@ -0,0 +1,46 @@
+package org.firstinspires.ftc.teamcode.opmodes;
+
+
+import com.qualcomm.robotcore.eventloop.opmode.LinearOpMode;
+import com.qualcomm.robotcore.eventloop.opmode.TeleOp;
+import com.qualcomm.robotcore.hardware.DcMotor;
+
+import org.firstinspires.ftc.teamcode.hardware.Drivebase;
+
+@TeleOp(name = "BasicTeleOp")
+public class BasicTeleOp extends LinearOpMode {
+    @Override
+    public void runOpMode() {
+        // ALWAYS SET TELEMETRY AUTOCLEAR to TRUE!!!
+        telemetry.setAutoClear(true);
+
+        telemetry.addData("Status", "Initializing");
+        telemetry.update();
+
+        Drivebase drivebase = new Drivebase(hardwareMap, null);
+
+        telemetry.addData("Status", "Initialized");
+        telemetry.update();
+
+        drivebase.clearEncoder(DcMotor.RunMode.RUN_WITHOUT_ENCODER);
+
+        waitForStart();
+
+        while (opModeIsActive()) {
+            double yInput = -gamepad1.left_stick_y;
```

Comment:
I think we should add an in-line comment describing why this is set to negative, indicating that the controller joystick has inversed y values.